DuckDuckGo/MainViewController.swift

```diff
@@ -721,4 +726,109 @@ extension MainViewController: Themable {
+            UIKeyCommand(input: UIKeyCommand.inputTab, modifierFlags: [.control, .shift], action: #selector(keyboardPreviousTab)),
+            UIKeyCommand(input: "]", modifierFlags: [.shift, .command], action: #selector(keyboardNextTab)),
+            UIKeyCommand(input: "[", modifierFlags: [.shift, .command], action: #selector(keyboardPreviousTab)),
+            UIKeyCommand(input: "]", modifierFlags: [.shift, .command], action: #selector(keyboardNextTab)),
+            UIKeyCommand(input: "[", modifierFlags: [.shift, .command], action: #selector(keyboardPreviousTab)),
+            UIKeyCommand(input: "\\", modifierFlags: [.shift, .control], action: #selector(keyboardShowAllTabs)),
+            UIKeyCommand(input: "]", modifierFlags: [.command], action: #selector(keyboardBrowserForward)),
+            UIKeyCommand(input: "[", modifierFlags: [.command], action: #selector(keyboardBrowserBack)),
+            UIKeyCommand(input: "f", modifierFlags: [.alternate, .command], action: #selector(keyboardFind)),
+            UIKeyCommand(input: UIKeyCommand.inputRightArrow, modifierFlags: [.command], action: #selector(keyboardBrowserForward)),
+            UIKeyCommand(input: UIKeyCommand.inputLeftArrow, modifierFlags: [.command], action: #selector(keyboardBrowserBack)),
+            UIKeyCommand(input: UIKeyCommand.inputUpArrow, modifierFlags: [], action: #selector(keyboardArrowUp)),
+            UIKeyCommand(input: UIKeyCommand.inputDownArrow, modifierFlags: [], action: #selector(keyboardArrowDown)),
+            UIKeyCommand(input: UIKeyCommand.inputEscape, modifierFlags: [], action: #selector(keyboardEscape))
+        ]
+    }
+    
+    @objc func keyboardFind() {
+        if let controller = homeController {
+            controller.launchNewSearch()
+        } else {
+            omniBar.becomeFirstResponder()
+        }
+    }
+    
+    @objc func keyboardEscape() {
+        
+        if let controller = autocompleteController {
+            controller.keyboardEscape()
+            homeController?.collectionView.omniBarCancelPressed()
+        } else if let controller = homeController {
+            controller.omniBarCancelPressed()
+        } else {
+            omniBar.resignFirstResponder()
+        }
+    }
+    
+    @objc func keyboardArrowDown() {
+        if let controller = autocompleteController {
+            controller.keyboardMoveSelectionDown()
+        } else {
+            currentTab?.webView.becomeFirstResponder()
+        }
+    }
+    
+    @objc func keyboardArrowUp() {
+        if let controller = autocompleteController {
+            controller.keyboardMoveSelectionUp()
+        } else {
+            currentTab?.webView.becomeFirstResponder()
+        }
+    }
+
+    @objc func keyboardNewTab() {
+        if let tab = currentTab {
+            tabDidRequestNewTab(tab)
+        } else {
+            keyboardFind()
+        }
+    }
+    
+    @objc func keyboardCloseTab() {
+        guard let tab = currentTab else { return }
```

Comment:
Feature request :) 
I wonder how hard would be to move back to last used tab in such scenario:

1. Have some tab open.
2. Press cmd+t to open new tab.
3. I'd love to use cmd+w to close home screen and enter last opened tab.

Reply:
Going to skip this one for now :)